.github/workflows/custom_docker_builds.yml

```diff
@@ -12,23 +12,21 @@ jobs:
         name: Checkout
         uses: actions/checkout@v2
       -
-        name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v1
-      -
-        name: Login to DockerHub
-        uses: docker/login-action@v1
+        name: Log in to the Container registry
+        uses: docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9
         with:
-          username: ${{ secrets.DOCKERHUB_USERNAME }}
-          password: ${{ secrets.DOCKERHUB_TOKEN }}
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
       -
         name: Build and push ci-bridge
         id: docker_build_sync
-        uses: docker/build-push-action@v2
+        uses: docker/build-push-action@ad44023a93711e3deb337508980b4b5e9bcdc5dc
```

Comment:
just curious, is this for security reasons?  Like you've vetted this sha as safe, and don't want them changing the action out from under you?

Reply:
These specific hashes were copied from the example documented here:

https://docs.github.com/en/packages/managing-github-packages-using-github-actions-workflows/publishing-and-installing-a-package-with-github-actions#publishing-a-package-using-an-action

Reply:
Ok, good to know.